HACKING.txt

```diff
@@ -25,6 +25,26 @@ tox on PyPI: https://pypi.org/project/tox/
 
   Alternatively, create a writeable fork on GitHub and clone it.
 
+- Make sure that `tox` is installed, either in your path, or locally. Examples
+  below assume that `tox` was installed with:
+
+    $ pip3 install --user tox
+    $ export TOX=$(python3 -c 'import site; print(site.USER_BASE + "/bin")')/tox
+
+  Feel free to use whatever your favourite method for installation is, one
+  popular one is `pipx`: https://github.com/pipxproject/pipx which then may be
+  easily added to your shell's PATH variable so you may execute tox without
+  providing a full path
```

Comment:
Break run-on sentence into a second sentence, line-wrapping as needed.

```
If you add `pipx` to your PATH, then you may execute `tox` without providing a full path when invoking it.
```

Reply:
Ugh, that's not clear. Should be delete "which then may be..." and replace with a complete sentence.